starter-code/css/main.css

```diff
@@ -6,3 +6,135 @@ Green: #00B172
 White: #FFF
 
 */
+html {
+    box-sizing: border-box;
+  }
+  *, *:before, *:after {
+    box-sizing: inherit;
+  }
+
```

Comment:
If you inspect your body tag, you'll notice that it has an ugly thin margin. Try to remove it by setting margin and padding 0